pangraph/Tuura/Pangraph/Main.hs

```diff
@@ -0,0 +1,29 @@
+module Tuura.Pangraph.Main (main) where
+
+import Control.Monad
+import System.FilePath
+
+import Tuura.Pangraph.Options
+import qualified Pangraph.Parser as P
+import qualified Pangraph.Writer as W
+
+main :: IO ()
+main = do
+    -- get arguments
+    options <- getOptions
+    let graphMLPath           = optGraphML options
+        graphVHDLPath         = (optResultDir options) </> "graph.vhdl"
```

Comment:
Why are these filenames fixed? It may be useful to set them from the command line.